lib/Net/DRI/DRD/GoDaddy/MZB.pm

```diff
@@ -0,0 +1,291 @@
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::LaunchPhase> urn:ietf:params:xml:ns:launch-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::IDN> urn:ietf:params:xml:ns:idn-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::AllocationToken> urn:ietf:params:xml:ns:allocationToken-1.0
+
+=head2 Custom extensions (NeuLevel)
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::NeuLevel::Message> urn:ietf:params:xml:ns:neulevel-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::NeuLevel::EXTContact> urn:ietf:params:xml:ns:neulevel-1.0 (For .NYC Only)
+
+=head3 L<NET::DRI::Protocol::EPP::Extensions::NeuLevel::WhoisType> urn:ietf:params:xml:ns:neulevel-1.0 (For .TEL Only)
+
+=head2 Custom extensions (ex-ARI .To use these, set profile type to 'epp_ari')
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::Application> urn:ar:params:xml:ns:application-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::Trademark> urn:ar:params:xml:ns:tmch-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::IDN> urn:ar:params:xml:ns:idn-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::Variant> urn:ar:params:xml:ns:variant-1.1
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::Price> urn:ar:params:xml:ns:price-1.2
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::KeyValue> urn:X-ar:params:xml:ns:kv-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::ExAvail> urn:ar:params:xml:ns:exAvail-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::ARI::Block> urn:ar:params:xml:ns:block-1.0
+
+=head1 SUPPORT
+
+For now, support questions should be sent to:
+
+E<lt>netdri@dotandco.comE<gt>
+
+Please also see the SUPPORT file in the distribution.
+
+=head1 SEE ALSO
+
+E<lt>http://www.dotandco.com/services/software/Net-DRI/E<gt>
+
+=head1 AUTHOR
+
+Paulo Jorge, E<lt>paullojorgge@gmail.comE<gt>
+
+=head1 COPYRIGHT
+
+Copyright (c) 2021 Paulo Jorge <paullojorgge@gmail.com>.
+All rights reserved.
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+See the LICENSE file that comes with this distribution for more details.
+
+=cut
+
+####################################################################################################
+
+sub new
+{
+ my $class=shift;
+ my $self=$class->SUPER::new(@_);
+ $self->{info}->{host_as_attr}=0;
+ $self->{info}->{contact_i18n}=4; ## LOC+INT
+
+ return $self;
+}
+
+sub periods  { return map { DateTime::Duration->new(years => $_) } (1..10); }
+sub name     { return 'GoDaddy::MZB'; }
+
+sub tlds     { return ('adultblock', 'adultblockplus'); }
+sub object_types { return ('mzb'); }
+sub profile_types { return qw/epp/; }
+
+sub transport_protocol_default
+{
+ my ($self,$type)=@_;
+
+ return ('Net::DRI::Transport::Socket',{},'Net::DRI::Protocol::EPP::Extensions::GoDaddy',{'default_product'=>'GODADDY_MZB'}) if $type eq 'epp';
```

Comment:
Actually is this GODADDY_MZB used anywhere?

Reply:
You're correct! This is not doing anything.
Another bad copy/paste from old Unireg EPS. For some reason at the time we added this because of a specific test linked to distinct servers/greetings. Odd since by what I remember even with Unireg they shared the same server. Maybe they have done some changes that I cannot remember.